fixture/ledger.properties.tmpl

```diff
@@ -0,0 +1,124 @@
+#
+# For ledger
+#
```

Comment:
I just copied this `.properties` file from https://github.com/scalar-labs/scalar/blob/v3.9.1/ledger/conf/ledger.properties.tmpl .